policy/policy.go

```diff
@@ -103,6 +103,14 @@ func (s Set) Get(p Policy) (string, bool) {
 	return v, ok
 }
 
+func (s Set) ToStringMap() map[string]string {
```

Comment:
This was added so that the root `flux` package doesn't have a dependency on `policy`, since this introduces a cycle